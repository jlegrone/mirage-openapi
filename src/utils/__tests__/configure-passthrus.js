import Mirage from 'mirage-config' // eslint-disable-line
import configurePassthrus from '../configure-passthrus'

const configs = [
  {
    definition: 'http://petstore.swagger.io/v2/swagger.json',
    namespace: 'http://petstore.swagger.io/v2',
  },
]
const server = new Mirage()
const handler = () => null

test('adds openAPI config url passthrough', () => {
  configurePassthrus({ configs, server, handler })
  expect(server.passthroughs).toMatchSnapshot()
})
