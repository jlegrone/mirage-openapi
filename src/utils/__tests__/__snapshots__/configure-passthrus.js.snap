// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adds openAPI config url passthrough 1`] = `Object {}`;
